hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenFreezeAccountHandler.java

```diff
@@ -125,7 +126,7 @@ private TokenRelation validateSemantics(
         validateTrue(account != null, INVALID_ACCOUNT_ID);
 
         // Check that the token is associated to the account
-        final var tokenRel = tokenRelStore.getForModify(accountId, tokenId);
+        final var tokenRel = tokenRelStore.getForModify(accountId, token.tokenId());
```

Comment:
Nit: I think using `tokenId` is fine. If `token.tokenId()` returns something else, we have a major issue anyway. 🙂

Reply:
Done.